nucliadb_protos/writer.proto

```diff
@@ -123,6 +124,9 @@ message BrokerMessage {
     repeated resources.FieldQuestionAnswerWrapper question_answers = 38;
 
     utils.Security security = 39;
+
+    repeated resources.ExtractedVectorsWrapper field_vectorsets = 40;
```

Comment:
@nuclia/learning this is a proposal for how you can send to nucliadb the vectors computed for different vectorsets.